UKHomeOffice/nmsw-ui
# NMSW-314 Redirect user to try again page if no email when reaches /your-details

# Ticket

NMSW-314

----

## AC

Given a user has reached /create-account/your-details
When we cannot identify their email address
Then we redirect them to the verification failed page

Given a user is on /create-account/verification-failed
When they click the resend verification email button
Then they are redirected to the /create-account/request-new-verification-link

----

## To test

- ensure you have a clean browser instance (open in new tab)
- run app
- go to /create-account/your-details in the browser
- > you should automatically be redirected to /create-account/verification-failed 
- > the page should show the attached screenshot
- click the Resend confirmation email button
- > you should be taken to  /create-account/request-new-verification-link

----

## Notes
